plugins/node.d/nutups_

```diff
@@ -43,7 +44,7 @@ voltages() {
 			echo "${i}.min 0"
 		done
 	else
-		upsc $UPS | sed -n '/volt/{
+		$UPSC $UPS | sed -n '/volt/{
```

Comment:
Please quote `"$UPSC"` in order to prevent whitespace issues.

See `shellcheck`, in case you want to take a look at more style issues :)